ccloud/docs/beginner-cloud.rst

```diff
@@ -264,23 +256,18 @@ Produce and consume records with Confluent Cloud CLI
 
       ccloud kafka topic consume demo-topic-1 -b
 
-#. Verify your output resembles:
+#. Verify your output resembles the following. It is expected to be out of order because of round-robin partitioner:
 
    .. code-block:: text
 
       Starting Kafka Consumer. ^C or ^D to exit
-      2
+      1
```

Comment:
Just to note, the start.sh script consumes numbers 1-10.

Reply:
@awalther28 good catch.  Dropping script down to 5 as well.